CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ find_package(tinyxml2_vendor REQUIRED)
 # Made available by the vendor package
 find_package(TinyXML2 REQUIRED)
 
+# Set VERSION from package.xml
+ament_package_xml()
```

Comment:
This will be automatically called later in `ament_package` see [1](https://github.com/ament/ament_cmake/blob/foxy/ament_cmake_core/cmake/core/ament_package.cmake#L21), [2](https://github.com/ament/ament_cmake/blob/foxy/ament_cmake_core/cmake/core/ament_package.cmake#L61), [3](https://github.com/ament/ament_cmake/blob/foxy/ament_cmake_core/cmake/core/ament_package_xml.cmake#L19)

Reply:
That's right, but I think we need to set the property targets first.

Reply:
I missed that we used it to set the target properties

Reply:
Namely here:
https://github.com/ros-planning/srdfdom/blob/705a24a49ce98519a174424879571a9119a2b146/CMakeLists.txt#L39-L40